CHANGELOG.md

```diff
@@ -4,6 +4,9 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
+## [1.5.0]
```

Comment:
Makes sense! Thanks for the feedback :)

Reply:
I would appreciate if you could change the `[1.5.0]` to `[Unreleased]` as the version is finalized before a new release is created.

Otherwise everything looks good, thank you for the contribution!